docs/LV_client/tutorials/Multi viewport.md

```diff
@@ -0,0 +1,24 @@
+# Configure multi viewports
+
+### Multi viewport
+Starting from release **1.12** the user can now configure secondary viewports. Modify the `"listen_port"` on the sensor configuration with any non-zero value to create additional viewports. User can modify the location and pose of the secondary viewports.
+
+Note: A value of zero on the `"listen_port"` is reserve for the main viewport. 
```

Comment:
Good note. Now the entire `ViewportCamera` sensor config is a special case to set the game viewport